packages/@uppy/companion/src/server/controllers/callback.js

```diff
@@ -1,8 +1,24 @@
 /**
  * oAuth callback.  Encrypts the access token and sends the new token with the response,
  */
+const serialize = require('serialize-javascript')
+
 const tokenService = require('../helpers/jwt')
 const logger = require('../logger')
+const oAuthState = require('../helpers/oauth-state')
+
+const closePageHtml = (origin) => `
+  <!DOCTYPE html>
+  <html>
+  <head>
+      <meta charset="utf-8" />
+      <script>
+      window.opener.postMessage(${serialize({ error: true })}, ${serialize(origin)})
```

Comment:
nit: serializing `{ error: true }` is not useful here.
```suggestion
      window.opener.postMessage({ error: true }, ${serialize(origin)})
```

Reply:
I think it's nice to have serialize there. It could help signal to a future developer that it is dangerous to add code here. If there is no `serialize` and in the future someone adds more data here without first adding `serialize`, then they could introduce a XSS vulnerability.